api/src/main/java/org/openmrs/Order.java

```diff
@@ -32,10 +32,18 @@
  * 
  * @version 1.0
  */
-public class Order extends BaseOpenmrsData implements java.io.Serializable {
+public class Order extends BaseOpenmrsData implements java.io.Serializable, Comparable{
 	
 	public static final long serialVersionUID = 4334343L;
-	
+
+	@Override
+	public int compareTo(Object o) {
```

Comment:
- It's good practice to check first if the passed in order is not null.
- Sequence number is an Integer which already has standard code for comparing integers that you don't need to duplicate, just delegate to the compare function of Integer class, so you won't need line 44 below that returns 0.